src/git.rs

```diff
@@ -11,8 +12,54 @@ pub fn git_fetch(root: &Path) -> Result<(), git2::Error> {
     Ok(())
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct BranchInfo {
+    name: String,
+    commit_hash: String,
+    time: i64,
+    message: Vec<String>,
+}
+
+fn get_branch_info(
+    branch_struct: Result<(Branch, BranchType), git2::Error>,
+) -> Result<BranchInfo, git2::Error> {
+    let branch = branch_struct?.0;
+    let lossy_name = String::from_utf8_lossy(branch.name_bytes()?);
+    let name = lossy_name
+        .strip_prefix("origin/")
+        .unwrap_or(&lossy_name)
```

Comment:
Should we just ignore anything that doesn’t start with `origin/` instead?

Reply:
this should in theory never happen because we clone it ourselves, if it does happen for whatever reason, it would be helpful to see it in the UI and then we'll know the issue vs having it disappear and then working through the stack to debug why a branch isnt showing up, but im not too fussed if you would like me to ignore it!